### PR TITLE
Added RDFUnit delay parameter

### PR DESCRIPTION
**Pull request by mclaurentiu**

### CODE REVIEW

File: validator/service_layer/handlers.py [hunk 1]
```diff
@@ -108,10 +109,23 @@ def run_sparql_endpoint_validator(sparql_endpoint_url: str, graphs_uris: List[st
 
     Please see https://github.com/AKSW/RDFUnit/wiki/CLI for a comprehensive description of the parameters
     """
+
+    load_dotenv()
+    try:
+        rdfunit_delay_ms = int(os.environ.get("RDFUNIT_DELAY_MS"), 10)
```
Comment (cosletmihai) [inline validator/service_layer/handlers.py:115]: we practice to "extract" the environment variables in through the use of a `config.py` file.

see example from [rdf differ](https://github.com/meaningfy-ws/rdf-differ/blob/master/rdf_differ/config.py)

Is our practice justified, judgind from your experience?

File: validator/service_layer/handlers.py [hunk 2]
```diff
@@ -108,10 +109,23 @@ def run_sparql_endpoint_validator(sparql_endpoint_url: str, graphs_uris: List[st
...
+    except ValueError:
+        logger.error("Invalid value for RDFUNIT_DELAY_MS: " + \
+                     str(os.environ.get("RDFUNIT_DELAY_MS")) + \
+                     ". Defaulting to 1 ms")
```
Comment (cosletmihai) [inline validator/service_layer/handlers.py:119]: I like this logging more and more!

**Review by cosletmihai — APPROVED** [review]

File: validator/service_layer/handlers.py (hunk 1 above)
Comment (cosletmihai) [reply validator/service_layer/handlers.py:115]: you could improve it by using the package you installed

File: validator/service_layer/handlers.py [hunk 3]
```diff
@@ -108,10 +109,23 @@ def run_sparql_endpoint_validator(sparql_endpoint_url: str, graphs_uris: List[st
...
+        rdfunit_delay_ms = 1
+
+
     logger.info("RDFUnitWrapper starting ...")
     validator_wrapper: AbstractValidatorWrapper
```
Comment (costezki) [inline validator/service_layer/handlers.py:124]: what does this line do? Out of curiosity, I have not seen this syntax before :)

**Review by costezki — APPROVED** [review]
well done!

**Pull request merged**